packages/auth0/src/handlers/auth0-handlers.ts

```diff
@@ -201,7 +201,31 @@ export const createAuth0Handlers = (store: Auth0Store, people: Iterable<Person>,
         idTokenData.nonce = nonce;
       }
 
-      let userData = {} as RuleUser;
+      /* userData per auth0 example
+        {
+          "name": "jdoe@foobar.com",
+          "email": "jdoe@foobar.com",
+          "user_id": "auth0|0123456789",
+          "nickname": "jdoe",
+          "picture": "http://foobar.com/pictures/jdoe.png",
+          "identities": [
+            {
+              "provider": "auth0",
+              "user_id": "0123456789",
+              "connection": "Username-Password-Connection",
+              "isSocial": false
+            }
+          ]
+        }
+      */
```

Comment:
I'm going to approve, but can you remove this before pushing?

Reply:
Sure, I can. I thought it might be nice to have an actual auth0 response, but perfectly fine removing it.